components/SearchBar.vue

```diff
@@ -131,6 +127,9 @@ const selectedLanguage: Ref<Locale | string> = ref('')
 
 const placeHolderTextDisplay = '----Any----'
 
+languageOptions[0].simpleText = placeHolderTextDisplay
```

Comment:
is this going to overwrite the language at `0`?

Reply:
you can use`[ `unshift()` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift)`for this

Reply:
There is no language at 0 in these display options. I thought time complexity wise this would be better than unshifting and shifting. It is written this way as for some components you want the first in a select field to be empty. This is a special case for the search bar. @ermish What do you think?